cmd/installer/cli/install.go

```diff
@@ -376,74 +477,57 @@ func preRunInstall(cmd *cobra.Command, flags *InstallCmdFlags, rc runtimeconfig.
 			flags.cidrCfg.GlobalCIDR = &installConfig.GlobalCIDR
 		}
 
+		flags.dataDir = installConfig.DataDirectory
 		flags.networkInterface = installConfig.NetworkInterface
 		flags.adminConsolePort = installConfig.AdminConsolePort
-		flags.dataDir = installConfig.DataDirectory
 		flags.localArtifactMirrorPort = installConfig.LocalArtifactMirrorPort
 
-	} else {
-		proxy, err := parseProxyFlags(cmd)
-		if err != nil {
-			return err
-		}
-		flags.proxy = proxy
-
-		if err := verifyProxyConfig(flags.proxy, prompts.New(), flags.assumeYes); err != nil {
-			return err
-		}
-		logrus.Debug("User confirmed prompt to proceed installing with `http_proxy` set and `https_proxy` unset")
-
-		if err := validateCIDRFlags(cmd); err != nil {
-			return err
+		if err := updateRuntimeConfigFromInstallCmdFlags(flags, rc); err != nil {
```

Comment:
rather than a configChan, we should just have a runtimeConfigChan and set it directly since its all computed in the api now.

Reply:
I got rid of the config chan in my branch that installs everything in the api, so there won't be a need for it